app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy(command: str) -> None:
+    cmd, current_file, future = command.split()
+    if current_file != future or cmd == "cp":
+        with open(current_file, "r") as file_in, open(future, "w") as file_out:
```

Comment:
Dont forget that you can cover everything in brackets (almost)
```suggestion
        with (open(current_file, "r") as file_in,
              open(future_file, "w") as file_out):
```